packages/reports/addon/components/report-actions/multiple-format-export.js

```diff
@@ -139,10 +164,41 @@ export default class MultipleFormatExport extends Component {
    */
   @action
   notify(type) {
+    const typeMessages = {
+      'Google Sheet': 'We are building your spreadsheet and sending to Google Drive.  Keep an eye out for the email!'
+    };
     this.naviNotifications.add({
-      message: `${type}? Got it. The download should begin soon.`,
+      message: typeMessages[type] ?? `${type}? Got it. The download should begin soon.`,
       type: 'info',
       timeout: 'medium'
     });
   }
+
+  @action
+  async handleAsync(isAsync, event) {
+    if (isAsync) {
+      const href = event.target.href;
+      event.preventDefault();
+      try {
+        const response = await fetch(href);
+        const json = await response.json();
+        this.naviNotifications.clearMessages();
```

Comment:
What happens if you transition away from a dashboard/report view before the fetch finishes? Will we get a  destroyed obj error when referencing `this.naviNotifications`?

Reply:
Can you show me a test to reproduce this?

Reply:
Would the changes I made fix this?

Reply:
> Would the changes I made fix this?
Yeah, I was thinking the same solution 👍 

> Can you show me a test to reproduce this?
1) Click export gsheet
2) Transition to report list view or somewhere else
3) Resolve gsheet promise;
4) No error, notification present.

Reply:
Yup it works 👍